Doc/library/dis.rst

```diff
@@ -42,11 +42,11 @@ the following command can be used to display the disassembly of
    >>> dis.dis(myfunc)
      1           0 RESUME                   0
 
-     2           2 PUSH_NULL
```

Comment:
Would it be possible to use doctest here ?

Reply:
I think we'd need to import `dis` to do that.

Reply:
I think we can do this at the top of the module:

```
.. testsetup::

   import dis
```

That's what [the itertools docs do](https://github.com/python/cpython/blob/3.10/Doc/library/itertools.rst).

Reply:
Fascinating, I had no clue that existed. TIL, thanks!